packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/action_validation/document_replace_transition_action/structure_v0/mod.rs

```diff
@@ -58,20 +58,18 @@ impl DocumentReplaceTransitionActionStructureValidationV0 for DocumentReplaceTra
         // Validate timestamps against block time
         // we do validation here but not in validate state because it's a cheap validation
         // and validate state implements expensive validation only
-        let latest_block_time_ms = platform.state.last_committed_block_time_ms();
+        let latest_block_time_ms = platform.state.any_block_info().time_ms;
         let average_block_spacing_ms = platform.config.block_spacing_ms;
 
-        if let Some(latest_block_time_ms) = latest_block_time_ms {
-            let validation_result = check_updated_inside_time_window(
-                self,
-                latest_block_time_ms,
-                average_block_spacing_ms,
-                platform_version,
-            )?;
+        let validation_result = check_updated_inside_time_window(
+            self,
+            latest_block_time_ms,
+            average_block_spacing_ms,
+            platform_version,
+        )?;
 
-            if !validation_result.is_valid() {
-                return Ok(validation_result);
-            }
+        if !validation_result.is_valid() {
+            return Ok(validation_result);
         }
```

Comment:
Regression here, please revert.

Reply:
It's a potential place for bugs. We always should have block time. We shouldn't change consensus if for some reason is not set.

Reply:
I will fix it in a separate PR